PixelHunter1995/GameManager.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using PixelHunter1995.GameStates;
+using PixelHunter1995.Inputs;
+
+namespace PixelHunter1995
+{
+    class GameManager
+    {
+        private StateManager StateManager;
+        private readonly SceneManager SceneManager;
+        public GameManager()
+        {
+            StateManager = new StateManager();
+            SceneManager = new SceneManager();
+        }
+
+        public void Initialize()
```

Comment:
not internal?

Reply:
Do we want it to be internal? Maybe rather make the rest public?